frontend/src/scenes/teamLogic.tsx

```diff
@@ -21,8 +28,14 @@ export const teamLogic = kea<teamLogicType<TeamType>>({
                 deleteTeamFailure: () => null,
             },
         ],
+        updatingTeamPayload: [
+            null as Partial<TeamType> | null,
+            {
+                setUpdatingTeamPayload: (_, { payload }) => payload,
+            },
+        ],
```

Comment:
Could this be taken directly from the `updateCurrentTeam` action?

Reply:
👍